### ISSUE TITLE
Missing method in VErtxOption

### ISSUE THREAD

**Issue by codepitbull** [vert-x3/vertx-codegen#157]

**Comment by vietj** [comment]
that's not really possible at the moment as data object methods only denotes vertxgen types - we may improve that in the future to open it to any type and have the shim to chose wether or not to expose these types

**Comment by codepitbull** [comment]
How about simply providing all **NOT** @VertxGen-annotated methods that are part of a public Vertx-API (so not impl-methods) as a separate collection?
Right now I have to manually handle them by adding specific handling in the templates. Getting a list of them would allow me to either log their existence or define some generic handling, in my case I'd simply pass them through as is without converting them.

**Comment by vietj** [comment]
yes, we should do that, I've added a section in my slides for the F2F meemting about it

it should also work for vertxgen types

**Comment by pmlopes** [comment]
Yes I believe that would be nice for groovy and ECMAScript too
